src/delphi/eval/calc_model_group_stats.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+
+def calc_model_group_stats(
+    tokenized_corpus_dataset: list,
+    logprob_datasets: dict[str, list[list[float]]],
+    token_groups: dict[int, dict[str, bool]],
```

Comment:
What we used to do at my old job with dicts was sth like
```suggestion
    logprobs_by_model: dict[str, list[list[float]]],
    labels_by_token: dict[int, dict[str, bool]],
```
maybe these aren't the best examples / names, but the idea was to describe what the key in a dict was. Sometimes recursively like x_by_y_by_z for a nested dict

Reply:
That's a nice convention, I'm in.